### ISSUE TITLE
Assign aria-level attribute to collection detail accordion headings

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/dimes#227]
## Describe the bug
Collection details need an `aria-level` attribute for accordion headings level 2.

## Expected behavior
The accordion uses `role="heading"` to assign headings, but is missing the `aria-level` attribute (should be level 2) for the "Summary" and "Access and Use" headings.
